src/components/views/PCIForm.js

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEqual } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import {
+  AccordionSet,
+  Button,
+  Col,
+  ExpandAllButton,
+  IconButton,
+  Layout,
+  Pane,
+  PaneFooter,
+  PaneMenu,
+  Paneset,
+  Row,
+  Spinner,
+} from '@folio/stripes/components';
+import { TitleManager } from '@folio/stripes/core';
+import stripesFinalForm from '@folio/stripes/final-form';
+import css from './PCIForm.css';
+
+import { PCIFormCoverage, PCIFormInfo } from '../EResourceSections';
+
+class PCIForm extends React.Component {
+  static propTypes = {
+    form: PropTypes.shape({
+      getRegisteredFields: PropTypes.func.isRequired,
+    }).isRequired,
+    handlers: PropTypes.PropTypes.shape({
+      onClose: PropTypes.func.isRequired,
+    }),
+    initialValues: PropTypes.object,
+    handleSubmit: PropTypes.func.isRequired,
+    pristine: PropTypes.bool,
+    submitting: PropTypes.bool,
+    values: PropTypes.object,
+  }
+
+  static defaultProps = {
+    initialValues: {},
+  }
+
+  state = {
```

Comment:
Same as the view, use AccordionStatus instead of controlling state yourself.

Reply:
Moved back to the old implementation as it seems like `AccordionStatus` doesnt go well with just a single `Accordion` child which is the case here.

Reply:
If we only have a single Accordion then we don't need any of that state handling because we don't need an expand-all button, though. Can just have a single uncontrolled Accordion.

(Can you let @JohnC-80 know about the bug you're running into?)